app/services/forest_liana/filters_parser.rb

```diff
@@ -47,28 +47,47 @@ def parse_aggregation(node)
     end
 
     def parse_condition(condition)
+      where = parse_condition_without_smart_field(condition)
+
+      if SchemaUtils.is_smart_field(@resource, field)
+        schema = ForestLiana.schema_for_resource(@resource)
+        field_schema = schema.fields.find do |field|
+          field.try(:name) == field_name
+        end
+
+        unless field_schema.try(:[], :filter)
```

Comment:
[MISC] I don't get it, you check for filter existence but don't use it afterward... 🤔

Reply:
Mistake I wrote `:search` while it was actually `:filter` 😅